src/app/shared/session-storage/session-storage.service.ts

```diff
@@ -0,0 +1,20 @@
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs/Rx';
+
+@Injectable()
+export class SessionStorageService {
+
+  private token: string;
+  constructor() {}
+
+  public getToken() {
```

Comment:
@gshigeto this isn't generic enough. It should be more like this:

```javascript
public getValue(key: string) {}

public setValue(key: string, value: string) {}
```

So you can then do the following: 

```javascript
this.storage.setToken('access_token', params['access_token']);
```

Then we'll be able to use this service more generically for other uses in session storage.